protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3312,6 +3326,72 @@ message ListEntitiesResponse {
   repeated vega.events.v1.ProtocolUpgradeEvent protocol_upgrade_proposals = 28;
 }
 
+// Request to list a market's funding periods
+message ListFundingPeriodsRequest {
+  // Market ID to get funding periods for.
+  string market_id = 1 [(google.api.field_behavior) = REQUIRED];
+  // Optional funding period sequence number to limit results to a specific funding period.
```

Comment:
would it be clear to someone using these APIs how to get the funding period sequence number? (This is a question for all the instances that mention this sequence number)

Reply:
I don't think so, but @edd has asked for a date range as input which makes much more sense so I'm just going to remove this option in the request